src/components/Header.tsx

```diff
@@ -0,0 +1,39 @@
+import logo from '../assets/logo.svg';
+import './Header.css';
+import useUser from '../hooks/useUser';
+
+interface User {
+  email: string;
+  image_source: string;

```

Comment:
```suggestion
  imageSource: string;
```
타입스크립트(자바스크립트)에서 타입 프로퍼티명도 객체명과 같이 카멜케이스를 사용합니다.